storage/sealing/types.go

```diff
@@ -26,9 +29,9 @@ type Log struct {
 }
 
 type SectorInfo struct {
-	State    api.SectorState
-	SectorID abi.SectorNumber
-	Nonce    uint64 // TODO: remove
+	State    SectorState
+	SectorID abi.SectorNumber // TODO: this field's name should be changed to SectorNumber
```

Comment:
Please do now, doing this later will be a breaking change (this struct is serialized and stored as a map[string])